docs/ldml/tr35-keyboards.md

```diff
@@ -1,8 +1,8 @@
-## Unicode Technical Standard #35 Tech Preview
+## Unicode Technical Standard #35
```

Comment:
thanks for catching this. I know i removed other wording but obviously missed this.